.github/workflows/wheel_image.yml

```
@@ -0,0 +1,72 @@
+name: Build custom Docker image for manylinux wheels
+
+on:
+  workflow_dispatch:
+    inputs:
+      base_image:
+        description: 'The base Docker image to use'
+        default: 'manylinux2014'
```

```suggestion
        default: 'manylinux_2_28'
```
This is the default since #3306